travisci/python-unittest_harness.sh

```diff
@@ -30,13 +32,20 @@ fi
 
 PYTEST_OPTIONS=()
 if [ "$COVERAGE" = 'true' ]; then
-  PYTEST_OPTIONS+=('--cov=./')
+  PYTEST_OPTIONS+=('--cov=./' '--cov-report=term-missing')
```

Comment:
This is confusing me, `PYTEST_OPTIONS=()` looks like a tuple, without comma separation? What is it?

Reply:
It is the way to write arrays in bash scripts. Quite confusing indeed :man_shrugging:

Reply:
:woman_facepalming: I was stuck in python